Apps/Sandcastle/gallery/Scene Rendering Performance.html

```diff
@@ -0,0 +1,283 @@
+function loadCzmlScenario() {
+    resetScene();
+
+    var czml = [{
+        "id" : "document",
+        "name" : "CZML Model",
+        "version" : "1.0"
+    }, {
+        "id" : "aircraft model",
+        "name" : "Cesium Air",
+        "position" : {
+            "cartographicDegrees" : [-77, 37, 10000]
+        },
+        "model": {
+            "gltf" : "../../SampleData/models/CesiumAir/Cesium_Air.glb",
+            "scale" : 2.0,
+            "minimumPixelSize": 128
+        }
+    }];
+
+    var dataSourcePromise = viewer.dataSources.add(Cesium.CzmlDataSource.load(czml));
+
+    dataSourcePromise.then(function(dataSource){
+        viewer.trackedEntity = dataSource.entities.getById('aircraft model');
+    });
+}
+
+// Pick an entity, only rendering when needed.
+function pickingScenario() {
+    resetScene();
+
+    var entity = viewer.entities.add({
+        position : Cesium.Cartesian3.fromDegrees(-75.59777, 40.03883),
+        box : {
+            dimensions : new Cesium.Cartesian3(1000000.0, 1000000.0, 30000.0),
+            material : Cesium.Color.CORNFLOWERBLUE
+        }
+    });
+
+    // If the mouse is over the box, change its scale and color,
+    // then request a new render frame.
+    var lastPicked;
+    handler = new Cesium.ScreenSpaceEventHandler(scene.canvas);
+    handler.setInputAction(function(movement) {
+        var pickedObject = scene.pick(movement.endPosition);
+        if (Cesium.defined(pickedObject) && (pickedObject.id === entity)) {
+            if (Cesium.defined(lastPicked)) {
+                return;
+            }
+
+            entity.box.material = Cesium.Color.YELLOW;
+            scene.requestRender();
+            lastPicked = pickedObject;
+        } else if (Cesium.defined(lastPicked)) {
+            entity.box.material = Cesium.Color.CORNFLOWERBLUE;
+            scene.requestRender();
+            lastPicked = undefined;
+        }
+    }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
+}
+
+// Changes to the scene with the API will require
+// calling requestRender() on change.
+function setScenePropertiesScenario() {
+    resetScene();
+
+    scene.skyBox.show = false;
+    scene.backgroundColor = Cesium.Color.CORNFLOWERBLUE;
+    scene.requestRender();
+}
+
+// BEGIN SANDCASTLE EXAMPLE UI SETUP
+
+var viewModel = {
+    requestRenderMode : true,
+    timeChangeEnabled : false,
+    maximumRenderTimeChange : 30.0,
+    lastRenderTime : '',
+    requestRender : function () {
+        scene.requestRender();
+    }
+};
+
+var toolbar = document.getElementById('toolbar');
+Cesium.knockout.track(viewModel);
+Cesium.knockout.applyBindings(viewModel, toolbar);
+
+Cesium.knockout.getObservable(viewModel, 'requestRenderMode').subscribe(function(value) {
+    scene.requestRenderMode = value;
+});
+
+Cesium.knockout.getObservable(viewModel, 'timeChangeEnabled').subscribe(function(value) {
+    scene.maximumRenderTimeChange = value ? viewModel.maximumRenderTimeChange : undefined;
```

Comment:
Why is the default for the `maximumRenderTimeChange` option passed to Viewer `Infinity` but this is being set to `undefined`?  Which is the right thing to do here?   From looking at the code, I guess it looks like they both do the same thing but I think it makes the example a little confusing.